src/System.CommandLine.Tests/System.CommandLine.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworkForNETSDK);$(NetFrameworkCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworkForNETSDK)</TargetFrameworks>
```

Comment:
@ViktorHofer I was getting the error `error CS0518: Predefined type 'System.Range' is not defined or imported` for usage of index and range syntax for `net481` within this project until I did this. I tried adding the classes directly as well as referencing Microsoft.Bcl.Memory and neither approach worked. 

Is this target framework needed for the tests? (It's not included for the main library.)

Reply:
The issue is in ApprovalTests: https://github.com/approvals/ApprovalTests.Net/issues/768

Reply:
We should definitely keep the .NET Framework tests. That validates that the library works correctly on modern .NET and .NET Framework.

Reply:
That would be great. Thanks!

Reply:
Done